spec/controllers/vacation_requests_controller_spec.rb

```diff
@@ -231,6 +231,13 @@
         end
       end
     end
+
+    it 'sends confirm vacation email notification' do
+      sign_in user
+
+      expect { send_request }
```

Comment:
Does it send notification for any request? What happens in case of:
- unauthorized access;
- DB error, like there is no such a request in DB.